package.json

```
@@ -7,19 +7,23 @@
     "type": "git",
     "url": "https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite.git"
   },
+  "type": "module",
   "scripts": {
-    "clean:node_modules": "pnpx rimraf --glob **/node_modules",
-    "clean:turbo": "turbo clean && turbo daemon stop && rimraf dist && rimraf .turbo",
-    "clean": "pnpm clean:turbo && pnpm clean:node_modules",
+    "clean-root:bundle": "pnpx rimraf dist && pnpx rimraf dist-zip",
+    "clean-root:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules",
+    "clean-root:turbo": "pnpx turbo daemon stop && pnpx rimraf .turbo",
+    "clean-root": "pnpm clean-root:bundle && pnpm clean-root:node_modules && pnpm clean-root:turbo",
+    "clean:node_modules": "pnpx turbo clean:node_modules && pnpm clean-root:node_modules",
+    "clean:turbo": "pnpx turbo clean:turbo && pnpm clean-root:turbo",
+    "clean:bundle": "pnpx turbo clean:bundle && pnpm clean-root:bundle",
+    "clean": "pnpx turbo clean && pnpm clean-root",
```

OK... There's too many clean scripts lol

I think in many case we need this clean scripts just for clean entire projects stuff. So I suggest leaving just the clean-root scripts.

```suggestion
    "clean:bundle": "pnpx rimraf dist && pnpx rimraf dist-zip",
    "clean:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules",
    "clean:turbo": "pnpx turbo daemon stop && pnpx rimraf .turbo",
    "clean": "pnpm clean-root:bundle && pnpm clean-root:node_modules && pnpm clean-root:turbo",
```

---

That suggestion isn't fully good, because of the mistake in `clean` but it's good change to make, i've done it.

I need to run turbo for the rest `clean:` scripts, check it and feel free to request more changes if needs :)